go.mod

```diff
@@ -2,28 +2,14 @@
 
 module k8s.io/test-infra
 
```

Comment:
As @airbornepony pointed out offline, it would be good to add a comment here saying "Please do not add a "replace" directive. See [LINK] for why they are bad for published packages." or similar.

Reply:
Done. I added some comments and also updated the dep.md doc.